doc/source/ray-core/ray-dag.rst

```diff
@@ -0,0 +1,155 @@
+.. _ray-dag-guide:
+
+Building Computation Graphs with Ray DAG API
+============================================
+
+With ``ray.remote`` you have the flexibility of running an application where
+computation is executed remotely at runtime. For a ``ray.remote`` decorated
+class or function, you can also use ``.bind`` on the body to build a static
+computation graph.
+
+.. note::
+
+     Ray DAG is designed to be developer facing API where recommended use cases
+     are
+     1) Locally iterate and test your application authored by higher level libraries.
+     2) Build libraries on top of the Ray DAG APIs.
+     Please see `DAG Unification Design Doc <https://docs.google.com/document/d/16wRobgsfCvOXnsaQq9lwQMDnnFWzXaYicIQ6e65W5W0>`_ for details.
```

Comment:
maybe also link the ray enhancement proposal? (there's one for serve right?)

Reply:
hmm i felt that one is very serve specific but this doc aims to be ray core level. Maybe link it at the bottom as additional resources ?

Reply:
make sense. link at the bottom is sensible